### PR TITLE
Only run useful hack scripts in branchff

### PR DESCRIPTION
**Pull request by caesarxuchao**

### CODE REVIEW

File: branchff [hunk 1]
```diff
@@ -157,8 +157,11 @@ logecho -n "Merging $MASTER_OBJECT into $RELEASE_BRANCH branch: "
 logrun -s git merge -X ours $MASTER_OBJECT
 
 # TODO: This is actually broken until #27992 lands
-logecho -n "update-all.sh: "
-logrun -s hack/update-all.sh
+logecho -n "Run useful hack/update* scripts: "
```
Comment (david-mcmahon) [inline branchff]:
```
# TODO: This should use update-all again when run-time returns to reasonable levels
logecho "The list of scripts might be stale. If not sure, run update-all.sh"
for script in openapi-spec generated-docs federation-openapi-spec; do
  logecho -n "Running $script: "
  logrun -s "hack/update-$script.sh"
done
```

**Comment by david-mcmahon** [comment]
This is of course the same problem we have running the release with the infamous hack/update-* scripts.

**Comment by caesarxuchao** [comment]
Incorporated the suggestion.

**Comment by david-mcmahon** [comment]
Thanks for sending this!
/lgtm

**Pull request merged**